src/scenes/Projects/StatusHistory/StatusWorkflowEvent.graphql

```diff
@@ -0,0 +1,29 @@
+fragment StatusWorkflowEvent on ProjectWorkflowEvent {
```

Comment:
```suggestion
fragment workflowEvent on ProjectWorkflowEvent {
```
"status" is meaningless here.

Reply:
`projectWorklofEvent` because `workflowEvent` was used by progress reports. That's why I used status in the first place.